content/en/database_monitoring/setup_postgres/alloydb.md

```diff
@@ -0,0 +1,367 @@
+
+
+[1]: /agent/docker/integrations/?tab=docker
+[2]: https://github.com/DataDog/integrations-core/blob/master/postgres/assets/configuration/spec.yaml#L417-L444
+[3]: /agent/configuration/secrets-management
+[4]: /agent/faq/template_variables/
+{{% /tab %}}
+{{% tab "Kubernetes" %}}
+
+If you have a Kubernetes cluster, use the [Datadog Cluster Agent][1] for Database Monitoring.
+
+Follow the instructions to [enable the cluster checks][2] if not already enabled in your Kubernetes cluster. You can declare the Postgres configuration either with static files mounted in the Cluster Agent container or using service annotations:
+
+### Helm
+
+Complete the following steps to install the [Datadog Cluster Agent][1] on your Kubernetes cluster. Replace the values to match your account and environment.
+
+1. Complete the [Datadog Agent installation instructions][3] for Helm.
+2. Update your YAML configuration file (`datadog-values.yaml` in the Cluster Agent installation instructions) to include the following:
+    ```yaml
+    clusterAgent:
+      confd:
+        postgres.yaml: -|
+          cluster_check: true
+          init_config:
+          instances:
+          - dbm: true
+            host: <INSTANCE_ADDRESS>
+            port: 5432
+            username: datadog
+            password: '<UNIQUE_PASSWORD>'
+            gcp:
+              project_id: '<PROJECT_ID>'
+              instance_id: '<INSTANCE_ID>'
+
+    clusterChecksRunner:
+      enabled: true
+    ```
+
+    For Postgres 9.6, add the following settings to the instance config where host and port are specified:
+
+    ```yaml
+    pg_stat_statements_view: datadog.pg_stat_statements()
+    pg_stat_activity_view: datadog.pg_stat_activity()
+    ```
+
+3. Deploy the Agent with the above configuration file from the command line:
+    ```shell
+    helm install datadog-agent -f datadog-values.yaml datadog/datadog
+    ```
+
+<div class="alert alert-info">
+For Windows, append <code>--set targetSystem=windows</code> to the <code>helm install</code> command.
+</div>
+
+[1]: https://app.datadoghq.com/organization-settings/api-keys
+[2]: /getting_started/site
+[3]: /containers/kubernetes/installation/?tab=helm#installation
+
+### Configure with mounted files
+
+To configure a cluster check with a mounted configuration file, mount the configuration file in the Cluster Agent container on the path: `/conf.d/postgres.yaml`:
+
+```yaml
+cluster_check: true  # Make sure to include this flag
+init_config:
+instances:
+  - dbm: true
+    host: '<INSTANCE_ADDRESS>'
+    port: 5432
+    username: datadog
+    password: '<PASSWORD>'
+    # After adding your project and instance, configure the Datadog GCP integration to pull additional cloud data such as CPU, Memory, etc.
```

Comment:
Same comment here -- is this line about GCP integration relevant for AlloyDB?

Reply:
Yes, as above